features/step_definitions/kata.rb

```diff
@@ -227,3 +227,78 @@
     end
   end
 end
+
+Given /^Verify catalog source existence$/ do
```

Comment:
@valtair1891 this needs admin check

Reply:
I checked that, and it don't 
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/210462/console

Reply:
`Given /^I verify catalog source existence$/ do`